tests/phpunit/tests/blocks/block.php

```diff
@@ -413,4 +413,120 @@ function test_passes_content_to_render_callback() {
 		$this->assertSame( 'abc', $block->render() );
 	}
 
+	/**
+	 * Tests `construct_wp_query_args` that is
+	 * helper function that constructs a WP_Query args
+	 * array from a `Query` block properties.
+	 *
+	 * @since 5.8.0
```

Comment:
All tests include the reference to their ticket. I don't think `since` is used in this context.

```suggestion
	 * @ticket 53240
```